src/BulletDynamics/Featherstone/btMultiBody.h

```diff
@@ -397,6 +402,26 @@ btMultiBody
 			m_deltaV[dof] += delta_vee[dof] * multiplier;
 		}
 	}
+    void applyDeltaSplitVeeMultiDof(const btScalar *delta_vee, btScalar multiplier)
+    {
+        for (int dof = 0; dof < 6 + getNumDofs(); ++dof)
+        {
+            m_splitV[dof] += delta_vee[dof] * multiplier;
+        }
+    }
+    void addSplitV()
+    {
+        applyDeltaVeeMultiDof(&m_splitV[0], 1);
```

Comment:
did you want to call applyDeltaSplitVeeMultiDof?
If calling  applyDeltaVeeMultiDof, it may change default behavior of the simulation.